sql/core/src/test/resources/sql-tests/results/charvarchar.sql.out

```diff
@@ -839,17 +839,26 @@ NULL	NULL	NULL	NULL
 -- !query
 select base64(c7), base64(c8), base64(v), ascii(s) from char_tbl4
 -- !query schema
-struct<base64(c7):string,base64(c8):string,base64(v):string,ascii(s):int>
+struct<>
 -- !query output
-NULL	NULL	NULL	NULL
-NULL	NULL	Uw==	NULL
-TiAgICAgIA==	TiAgICAgICA=	TiA=	78
-TmUgICAgIA==	TmUgICAgICA=	U3A=	78
-TmV0ICAgIA==	TmV0ICAgICA=	U3BhICA=	78
-TmV0RSAgIA==	TmV0RSAgICA=	U3Bhcg==	78
-TmV0RWEgIA==	TmV0RWEgICA=	U3Bhcmsg	78
-TmV0RWFzIA==	TmV0RWFzICA=	U3Bhcms=	78
-TmV0RWFzZQ==	TmV0RWFzZSA=	U3Bhcmst	78
+org.apache.spark.sql.AnalysisException
```

Comment:
ditto. This seems to be a wrong regeneration due to the improper implementation.